### DISCUSSION TITLE
Difference between creating separate r-universes

### DISCUSSION THREAD

**Discussion by jhk0530** [r-universe-org/help#456] [Q&A] [answered]
Hi, thanks for cool work. 
After introduction in useR! 2024, I'm going to follow these again.

I think this may be silly question, but what is main difference with building personal r-universe as `<username>.r-universe.dev` repository. (let me call this approach as **repository**)

Because even if user didn't make **repository** which contains `packages.json`, r-universe already built personal r-universe page automatically and it seems not much differ with **repository** way.

So My concern is that, if I recommend to create `<username>.r-universe.dev` to colleague (which is super cool I think) but they said, "Why I have to do that?"  and I don't have idea to this. 

I'm assuming this is for sharing R packages in other ways than github, (or adding some articles) but I'm not sure, so I thought I'd ask.

Thanks !

**Comment by jeroen** [comment, marked as answer]
You only need a personal `packages.json` if you want to add packages to r-universe which are not on CRAN or BioConductor.

**Reply by jhk0530** [reply]
Oh, I get it now, thank you very much.
